keras_cv/layers/preprocessing/random_height.py

```diff
@@ -0,0 +1,129 @@
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing.base_image_augmentation_layer import (
+    BaseImageAugmentationLayer,
+)
+from keras_cv.utils import preprocessing
+
+BILINEAR = "bilinear"
+NEAREST = "nearest"
+BICUBIC = "bicubic"
+AREA = "area"
+LANCZOS3 = "lanczos3"
+GAUSSIAN = "gaussian"
+MITCHELLCUBIC = "mitchellcubic"
+HEIGHT_AXIS = -3
+WIDTH_AXIS = -2
+IMAGES = "images"
+BOUNDING_BOXES = "bounding_boxes"
+
+class RandomHeight(BaseImageAugmentationLayer):
+    """A preprocessing layer which randomly varies image height during training.
```

Comment:
I *think* the canonical description would start with a one-liner describing what the layer does, and then blank, and then another paragraph on what is the behavior of this layer

Reply:
Actually, this is a direct copy of the KPL docstring. I'll customize it out specific to KerasCV

Reply:
@suvadityamuk is this now resolved?  Make sure to click "resolve conversation" after fixing issues!